api/src/main/java/org/openmrs/Obs.java

```diff
@@ -1301,6 +1347,9 @@ public Interpretation getInterpretation() {
 	/**
 	 * @since 2.1.0
 	 */
+	@Access(AccessType.PROPERTY)
```

Comment:
This is already mapped in the superclass

Reply:
True.I have noticed.